packages/lodestar/src/chain/banners/pandaFinalized.ts

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable no-useless-escape */
+export const POS_PANDA_FINALIZED_BANNER = `
```

Comment:
@philknows , do we want a similar banner on the finalization of the `merge`, i.e. first finalized checkpoint post merge?

Reply:
I think just the merge transition block is enough for the banner. It's what the other clients are doing. Finalization can just remain a normal log statement.